package.json

```diff
@@ -10,6 +10,7 @@
     "build": "nuxt generate",
     "dev:electron": "cross-env ELECTRON=true nuxt --port 9001 dev",
     "build:electron": "cross-env ELECTRON=true nuxt generate",
+    "package:electron": "node node_modules/electron-builder/cli.js",
```

Comment:
I'd prefer using the command `electron-builder` (it's inside a npm command context, so no need to prefix with `npx`) here instead of referencing the script the package `electron-builder` defines as `bin`. Not much of a change though, but rather a thing of flexibility...

Reply:
Ah, I didn't know that the npm context worked like that. If that works, it's much better.

Reply:
Yep, changed it, and it works nicely.